src/Microsoft.DotNet.Build.Tasks.net45/ExtractResWResourcesFromAssemblies.cs

```diff
@@ -72,6 +85,25 @@ public void CreateReswFiles()
                 }
             }
         }
+
+        private bool ShouldExtractResources(string expectedReswFileName, string assemblyPath)
```

Comment:
It will just not find the file and return true for the first check: https://github.com/dotnet/buildtools/pull/1630/files#diff-4dba46151dfcc864a9186f4ae3de64c1R48

But then when trying to load the assembly it will fall into the `catch (BadImageFormatException)` and will ignore it and continue to the next assembly.